code-of-conduct-2024-addendum.md

```diff
@@ -0,0 +1,27 @@
+# Process notes and drafted changes, for vote in November 2024:
+
+## Changes to new joiner messaging
+
+The following should be read to every new member when they join, in addition to the existing practices and guidelines that appear when a key card is activated:
+
+* No Harassment: This includes, but is not limited to, attention that comes after a request to stop. Our anti-harassment policy is part of our member agreement - please read it.
+* No Bullying: Recognize and respect everyone’s differences. Treat all members with kindness and respect.
+
+
+## New procedure/form for code of conduct incident reporting
+
+Board members should collect the following standard information when receiving an incident report. This may be an online form in the future but isn't being added as one as a part of this proposal.
+
+* Uniquely identifiable name of the reporter (Slack handle or full name)
+* Date/time of report
+* How would you like us to contact you about this concern/incident?
+* Confidentiality: We strive to keep all incidents confidential within the board, and to follow up with you before any follow up actions we take that could risk reidentifying you to a harasser. Is there any additional concern you have, or special handling that you need to request? (e.g. if a board member caused the incident, or if there is a time-sensitive concern)
+* Details of the issue/incident:
+  * Who?
+  * What?
+  * When?
+  * Where?
+  * How?
+* If there is any action you'd like to request that the board takes in response to this specific incident - let us know here.
```

Comment:
I would be clear that just because something is requested doesn't mean that the board will do that thing. If this is level setting from the side of the person reporting the incident to the board, then there should be some level setting in the other direction to prevent hurt feelings.

Reply:
I agree with that point and that there should be level-setting. I imagined that level-setting being more of an in-person discussion - it feels kind of discouraging to me to say "Hey, what do you want to happen, btw we aren't promising not to ignore what you want if we have other obligations that require us to take specific actions" during the initial report - but maybe it's better to have it be known in advance since it's ultimately true anyway? There's probably some way to say it that doesn't discourage reporting?

Reply:
I thought about this more, I think it is better to be transparent in advance that while the intent is to respect a reporter's autonomy, not all requested actions are possible. Edited, resolving this but please let me know if it isn't resolved.